src/rpc/handlers/LedgerEntry.cpp

```diff
@@ -157,17 +157,24 @@ LedgerEntryHandler::process(LedgerEntryHandler::Input input, Context const& ctx)
         return Error{*status};
 
     auto const lgrInfo = std::get<ripple::LedgerHeader>(lgrInfoOrStatus);
-    auto const ledgerObject = sharedPtrBackend_->fetchLedgerObject(key, lgrInfo.seq, ctx.yield);
-
-    if (!ledgerObject || ledgerObject->empty())
-        return Error{Status{"entryNotFound"}};
+    auto output = LedgerEntryHandler::Output{};
+    output.deletedLedgerIndex = std::nullopt;
```

Comment:
This line is redundant - the line above already creates `Output` with a `nullopt` for this field via `optional`'s default constructor.